nyc_taxis/operations/default.json

```diff
@@ -959,15 +959,16 @@
     {
       "name": "avg_passenger_count_esql_segment_partitioning",
       "operation-type": "esql",
-      "query" : "FROM nyc_taxis | stats avg(passenger_count)",
-      "body": { "pragma": { "data_partitioning": "segment" } }
+      "query" : "FROM nyc_taxis | stats avg(passenger_count)"
```

Comment:
Nice. The removal of the body and the serverless macros makes this much easier to read. We can still see that it is `segment_partitioning` in the `name`, so that knowledge is there for future readers of this code.

Reply:
That is a great point.